subsys/nfc/lib/platform.c

```diff
@@ -51,6 +63,87 @@ nrfx_err_t nfc_platform_setup(void)
 }
 
 
+static nrfx_err_t nfc_platform_tagheader_get(u8_t index, u32_t *tag_header)
+{
+	const u32_t tag_header_addresses[] = {
+		(u32_t) &NRF_FICR->NFC.TAGHEADER0,
+		(u32_t) &NRF_FICR->NFC.TAGHEADER1,
+		(u32_t) &NRF_FICR->NFC.TAGHEADER2
+	};
+
+#if CONFIG_TRUSTED_EXECUTION_NONSECURE
+	int err;
+
+	err = spm_request_read(tag_header, tag_header_addresses[index],
+			       sizeof(u32_t));
+	if (err != 0) {
+		LOG_ERR("Could not read NFC Tag Header FICR (err: %d)", err);
+		return NRFX_ERROR_INVALID_ADDR;
+	}
+#else
+	*tag_header = *((u32_t *) tag_header_addresses[index]);
+#endif
+
+	return NRFX_SUCCESS;
+}
+
+
+nrfx_err_t nfc_platform_nfcid1_default_bytes_get(u8_t * const nfcid1_buff,
+						 u32_t        nfcid1_buff_len)
+{
+	u32_t nfc_tag_header[3];
+
+	if ((nfcid1_buff_len != NRFX_NFCT_NFCID1_SINGLE_SIZE) &&
+	    (nfcid1_buff_len != NRFX_NFCT_NFCID1_DOUBLE_SIZE) &&
+	    (nfcid1_buff_len != NRFX_NFCT_NFCID1_TRIPLE_SIZE)) {
+		return NRFX_ERROR_INVALID_LENGTH;
+	}
+
+#if defined(FICR_NFC_TAGHEADER0_MFGID_Msk)
+	nrfx_err_t err_code;
```

Comment:
Please use `err` instead of `err_code`. This is what is used in this repo.

```
➜  nrf git:(75eb77589c) ✗ rg -t c "int err_code" subsys samples lib | wc
      2       7     129
➜  nrf git:(75eb77589c) ✗ rg -t c "int err" subsys samples lib | wc 
    485    1764   26593
➜  nrf git:(75eb77589c) ✗ 

```

Reply:
fixed